crates/aptos/src/node/mod.rs

```diff
@@ -184,26 +158,8 @@ impl CliCommand<Transaction> for IncreaseLockup {
     }
 
     async fn execute(mut self) -> CliTypedResult<Transaction> {
-        if self.lockup_duration.is_zero() {
-            return Err(CliError::CommandArgumentError(
-                "Must provide a non-zero lockup duration".to_string(),
-            ));
-        }
```

Comment:
What happened to the lockup duration?

Reply:
I'm just the messenger...

But the new model just increases the lockup to the current maximum.